selenium/che-selenium-test/src/test/resources/conf/selenium.properties

```diff
@@ -28,11 +28,11 @@ google.password=
 # Define configuration for che default test user.
 # For che single user assembly, the default values will be enough.
 # For che multiuser assembly, user must be already created on identity provider service.
-che.test_user.email=che@eclipse.org
-che.test_user.password=secret
+che.test_user.email=test@gmail.com
+che.test_user.password=password
 
 # Define configuration for admin test user.
 # Note that admin user needs only for multiuser assembly and
 # must be already created on identity provider service
-che.admin_user.email=
-che.admin_user.password=
+che.admin_user.email=admin@admin.com
```

Comment:
From security reasons it's better not to place real credentials in the public config file.